finetune/model.py

```diff
@@ -339,10 +368,10 @@ def _construct_graph(self, n_updates_total, n_classes, train=True):
                 features_aggregator.append(featurizer_state['features'])
 
                 if n_classes is not None:
-                    classifier_state = classifier(
+                    classifier_state = self.feat_to_logits(
```

Comment:
`classifier_state` and `feat_to_logits` are a little bit misleading because the terminology doesn't translate to regression models